src/components/elements/Layout.jsx

```diff
@@ -1,20 +1,21 @@
-import { useContext } from 'react'
+import { useContext, useState } from 'react'
 import { useNavigate, useLocation } from 'react-router-dom'
 import LocaleContext from 'utils/LocaleContext'
 import i18n from 'utils/i18n'
-import { ReactComponent as SmallLogo } from 'images/logo-small.svg'
 import 'components/elements/Layout.css'
 
 function Layout(props) {
   const { locale } = useContext(LocaleContext)
   const navigate = useNavigate()
   const location = useLocation()
-  const renderHeaderLogo = location.pathname !== '/'
+  const [selected, setSelected] = useState('true')
```

Comment:
Let's use the existing local global react context to set/get locale, 
```
// utils/LocaleContext.js

import React from "react";

const defaultValue = {
  locale: 'en',
  setLocale: () => {}
}

export default React.createContext(defaultValue);
```

We can change the setter to use localStorage and we can change the getter to have a default value of english if there is no item present inside of localStorage for language